save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/Vulnerability.kt

```diff
@@ -0,0 +1,45 @@
+package com.saveourtool.save.entities.vulnerabilities
+
+import com.saveourtool.save.entities.vulnerability.VulnerabilityDto
+import com.saveourtool.save.spring.entity.BaseEntityWithDto
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+
+import javax.persistence.*
+
+/**
+ * @property name
+ * @property progress
+ * @property description
+ * @property projects
+ **/
+@Entity
+class Vulnerability(
+
+    var name: String,
+
+    var progress: Int,
+
+    @OneToOne
+    @JoinColumn(name = "description_id")
+    var description: VulnerabilityDescription?,
```

Comment:
Still not sure if I understand your idea right. You say that description can be to have so that you do not want to store it in the same place with all the other `Vulnerability` info. But is there any difference between your implementation and the simple one (storing `description` as a `String`)? Sorry for my stupid questions, just trying to get the idea

Reply:
Maybe I'll remove this field from here or change it to Lazy fetch type.